packages/shared/src/styles/components/buttons.css

```diff
@@ -261,3 +261,21 @@
     --button-icon: var(--button-hover-icon);
   }
 }
+
+.btn-signback {
```

Comment:
Tried to do it using the Button component, but there were **a lot** of overriding classes and wrapping of components. Which is way more complicated than creating one for the sign back in button.

Reply:
Hmm that's sad :( 
Maybe once we refactor the button

Reply:
Yeah, hopefully soon, then we can also do the input elements. Lots of things are going on in those components.